### PR TITLE
Alt text support

### PR DESCRIPTION
**Pull request by whatisjasongoldstein**
$ manage migrate cropduster

### CODE REVIEW

File: cropduster/templatetags/cropduster_tags.py [hunk 1]
```diff
@@ -59,6 +60,14 @@ def get_crop(image, crop_name, exact_size=False, **kwargs):
     data = {}
     data['url'] = getattr(Image.get_file_for_size(image, crop_name), 'url', None)
 
+    if image.related_object:
```
Comment (fdintino) [inline cropduster/templatetags/cropduster_tags.py]: Why not move this stuff up here? I think we probably want to deprecate `exact_size`, but that should probably be a separate commit, no?

File: cropduster/templatetags/cropduster_tags.py [hunk 2]
```diff
@@ -59,6 +60,14 @@ def get_crop(image, crop_name, exact_size=False, **kwargs):
...
+        data.update({
+            "attribution": image.related_object.attribution,
+            "attribution_link": image.related_object.attribution_link,
+            "caption": image.related_object.caption,
+            "alt": image.related_object.alt_text,
```
Comment (fdintino) [inline cropduster/templatetags/cropduster_tags.py]: For consistency perhaps the key should also be `alt_text` as well

Comment (whatisjasongoldstein) [inline cropduster/templatetags/cropduster_tags.py]: I made it alt for cleanliness in the template.

**Comment by fdintino** [comment]
Be mindful that force pushing multiple times too quickly causes Travis to freak out.
